webapp/graphite/render/functions.py

```diff
@@ -861,6 +871,7 @@ def asPercent(requestContext, seriesList, total=None, *nodes):
   Param('total', ParamTypes.seriesList),
   Param('nodes', ParamTypes.nodeOrTag, multiple=True),
 ]
+asPercent.aggregator = True
```

Comment:
This doesn't return a single series, so probably doesn't make sense as an aggregator.  It will "work" but for each group you'd end up with the percentage of the group total represented by the first series in the group.